fili-core/src/test/groovy/com/yahoo/bard/webservice/util/TableUtilsSpec.groovy

```diff
@@ -85,8 +85,10 @@ class TableUtilsSpec extends  Specification {
         request.filterDimensions >> ds1
         query.metricDimensions >> ds1
         query.dependentFieldNames >> ([metric1, metric2, metric3] as Set)
+        PhysicalTable physicalTable = new PhysicalTable("", DefaultTimeGrain.DAY.buildZonedTimeGrain(DateTimeZone.UTC), [:])
+        0 * physicalTable.getPhysicalColumnName(_)
```

Comment:
This assertion doesn't make sense to me, for a couple of reasons:

1. Why are we adding the assertion? We didn't have it before, what value does adding it provide? <sub>Note: I'm not saying we _shouldn't_ add it, I just was curious why we might want to</sub>
2. Why does this test pass now? It looks like `TableUtils::getColumnNames` _should_ be calling `getPhysicalColumnName` for _dimensions_, so I think it should be called at least _once_. Something is very fishy here.